search.go

```diff
@@ -633,55 +633,56 @@ func unpackAttributes(children []*ber.Packet) []*EntryAttribute {
 }
 
 // DirSync does a Search with dirSync Control.
-func (l *Conn) DirSync(searchRequest *SearchRequest, flags int64, maxAttrCount int64, cookie []byte) (*SearchResult, error) {
-	var dirSyncControl *ControlDirSync
```

Comment:
I think DirSync() method has an unneeded code/process for some reason. Could you review and confirm whether my changes with no side effects?

Reply:
lgtm